bucket_api.go

```diff
@@ -78,7 +78,6 @@ func (bucket *Bucket) CloseAndDelete() (err error) {
 	defer bucket.mutex.Unlock()
 	if bucket.url != "" {
 		err = DeleteBucketAt(bucket.url)
-		bucket.url = ""
```

Comment:
It looks like this acts as protection against calling DeleteBucketAt for an already deleted bucket, in the case CloseAndDelete is called more than once.